app/controllers/permissions_controller.rb

```diff
@@ -361,6 +354,9 @@ def construct_request_object(provider)
       end
 
       collection_identifier['entry_titles'] = entry_titles
+      if params[:hidden_collections]
+        collection_identifier['entry_titles'] += params[:hidden_collections].split(';;; ')
```

Comment:
Let's talk about a different way of doing this, delimiters like this are bad news bears.